TEveEventDisplay/src/dict_classes/StyleOpts.hh

```diff
@@ -0,0 +1,3 @@
+const Int_t transpOpt = 100;
+const Int_t linewidth =  3; 
+const Int_t fontsize = 15; 
```

Comment:
You define these variables at file scope here and also in other files.  That's either illegal in C++ or will produce unexpected behaviour.   I presume that you have not yet had a problem because this file is not yet included anywhere.  

For the record, here are the files that contain definitions of these variables:
TEveEventDisplay/src/TEveMu2eTracker.cc
TEveMu2e_base_classes/TEveMu2eMCInterface.h
TEveEventDisplay/src/TEveMu2eMCInterface.cc

At a minimum, make the content of this file a struct with public data members.  That provides scope protection.  You may also want to make them constexpr, not just const.  That depends on future plans that we can discuss after this PR is in.

After this PR we should talk about future plans.  We talked earlier about being able to save a configuration?  Are these details part of the configuration that a use might want to modify and save?

Reply:
This is a work in progress, as always the TEve code is iterative, this PR is a transitional one, once I have all the numbers I need and understand the overlaps this will be used. Expect the next iteration in the next few days. The purpise of the PR is to get the FCL files in for people to use